WordPress/Classes/ViewRelated/Aztec/ViewControllers/AztecPostViewController.swift

```diff
@@ -2292,7 +2292,7 @@ extension AztecPostViewController {
         case .processing:
             DDLogInfo("Creating media")
         case .thumbnailReady(let url):
-            handleThumbnailURL(url, attachment: attachment)
+            handleThumbnailURL(url, attachment: attachment, then: { self.post.content = self.getHTML() })
```

Comment:
Shouldn't this be [weak self] in the block?

Reply:
I don't think `weak` is necessary in this case since we want to make sure that `self` still exists by the time the closure is called I (which should be right away).

Reply:
If you want to make sure it exists  you can do `[weak self]
if let self = self {
}`

Otherwise you're risking retention cycle

Reply:
I changed the implementation to keep the consistency of one-line calls inside this `switch` statement. Let me know what you think.